canopen/node/remote.py

```diff
@@ -67,3 +83,28 @@ def restore(self, subindex=1):
             4 - 127 = Manufacturer specific
         """
         self.sdo.download(0x1011, subindex, b"load")
+
+    def load_configuration(self):
```

Comment:
This method could actually be in the BaseNode class. The only thing is that you can skip the access type check on a local node as writes will always succeed there.

Reply:
I don't know about this, the ``load_configuration()`` is responsible for the upload of the OD (loaded from the .eds file) to the remote node trough SDO's calls, if the node is local the OD has already the values defined since they were stored in the OD when the .eds file was read.

Reply:
Yes you're right. I just need to add support for reading ParameterValue before reading DefaultValue in the local node.

Reply:
Yes, if the ``ParamterValue`` is set it is stored in the ``value`` variable, otherwise the ``value`` variable should hold the ``DefaultValue``. I have not implemented this at the OD level, for both **local** and **remote**, because I wanted to load values only defined in the ``ParamterValue`` to the **remote** node and not the default values, because the remote node could already have values defined and I don't want to overwrite this with the default values.